src/PowerTools/source.extension.vsixmanifest

```diff
@@ -14,35 +13,23 @@ When right-clicking on a file containing a derived DbContext class, the followin
 1) View Entity Data Model XML - Displays the EDMX XML representing the underlying Code First model.
```

Comment:
Nit: is this is in sync with the options actually displayed? See https://github.com/aspnet/EntityFramework6/blob/master/src/PowerTools/DbContextPackage.vsct